package.json

```diff
@@ -43,10 +43,11 @@
     "private::test.standalone": "cypress run --browser chrome --spec 'tests/cypress/tests/standalone/**/*'",
     "build.watch": "webpack-dev-server --content-base dash_table --mode development --config webpack.dev.config.js",
     "build": "run-s private::build:js private::build:py",
+    "format": "run-s \"private::lint:ts -- --fix\"",
     "lint": "run-s private::lint:*",
     "test.server": "run-p --race private::host* private::test.server",
     "test.standalone": "run-p --race private::host_js private::test.standalone",
-    "test.unit": "run-s private::test.python private::test.unit",
+    "test.unit": "run-s build private::test.python private::test.unit",
```

Comment:
removing the baseline artifacts, turns out the Py unit tests were always running against the latest release.. oops!

Reply:
but if you run this locally you've probably already built, so would be annoying to have the test command do it again. Could we do this in the CI config instead?

Reply:
https://github.com/plotly/dash-table/pull/603/commits/2adceab9c9fe8fd9803eb20b3ee9841fe152b372